compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java

```diff
@@ -321,48 +368,96 @@ private static boolean isCallToAnyStringBuilderMethod(AbstractInsnNode insn) {
                 && ((MethodInsnNode) insn).owner.equals(Type.getInternalName(StringBuilder.class));
     }
 
-    private static AbstractInsnNode convertConstructorCall(TypeInsnNode typeInsn,
-            MethodNode ctorMethod, ClassNode owner, NeoMethod callingNeoMethod,
-            CompilationUnit compUnit) throws IOException {
+    private static AbstractInsnNode convertStructConstructorCall(TypeInsnNode typeInsnNode, MethodNode ctorMethod,
```

Comment:
I'm pretty sure that parts of this can be refactored by using functionality from the `MethodConvertor`, i.e., passing the INVOKESPECIAL-MethodInsn to the `MethodConverter` and letting it do the work. Though, we can separate refactoring into another issue and move on with this one.

Reply:
Will be handled in #741.